PaddleCV/image_classification/utils/optimizer.py

```diff
@@ -123,8 +151,11 @@ def __init__(self, args):
         self.momentum_rate = args.momentum_rate
         self.step_epochs = args.step_epochs
         self.num_epochs = args.num_epochs
-
+        self.warm_up_epochs = args.warm_up_epochs
+        self.decay_steps = args.decay_steps
```

Comment:
confused with self.step_epochs and self.warm_up_epochs, self.decay_steps

Reply:
change decay_steps to decay_epochs, and add comment in readme.